common/src/claims.rs

```diff
@@ -135,6 +135,13 @@ impl ScopeBuilder {
             Scope::DeploymentPush, // To start an idle deploy
             Scope::Resources,      // To get past resources for an idle deploy
             Scope::Service,        // To get the running deploy for a service
+            // To add the locally persisted resources from the older deployers
+            // to the resource-recorder the first time the old deployer is updated
+            // to the new deployer (based on resource-recorder). After this moment
+            // all requests are routed to the resource-recorder instead of local
+            // persistence. We also use it when new project resources are provisioned
+            // and must be saved to the resource-recorder (for new deployments).
```

Comment:
Hmm, won't new deployments have a token/claim from the user? I may be out of the loop here, though.

Reply:
Hmm, you're actually right. 🤦 This is for the deployer permissions. Removed the clarifying note. Thanks :D